### PR TITLE
Make idempotency check a proper vulpix test

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: compiler/test/dotty/tools/dotc/CompilationTests.scala [hunk 1]
```diff
@@ -346,13 +290,9 @@ class CompilationTests extends ParallelTesting {
     assert(new java.io.File("../out/idempotency1/").exists)
     assert(new java.io.File("../out/idempotency2/").exists)
 
-    val t0 = System.currentTimeMillis()
-    checkIdempotency()
-    println(s"checked bytecode idempotency (${(System.currentTimeMillis() - t0) / 1000.0} sec)")
+    compileFile("../tests/idempotency/IdempotencyCheck.scala", defaultOptions).checkRuns()
```
Comment (smarter) [inline compiler/test/dotty/tools/dotc/CompilationTests.scala]: Why is this better?

Comment (felixmulder) [inline compiler/test/dotty/tools/dotc/CompilationTests.scala]: Because of reasons

Comment (felixmulder) [inline compiler/test/dotty/tools/dotc/CompilationTests.scala]: It improves the feedback from error messages for the venerable Stucki :)

Comment (nicolasstucki) [inline compiler/test/dotty/tools/dotc/CompilationTests.scala]: Error messages and timing is handled by vupix. If there is a crash, vulpix will report it and continue running the test suite.

Comment (smarter) [inline compiler/test/dotty/tools/dotc/CompilationTests.scala]: But why would there be a crash in the testcode?

**Pull request merged**